src/routes/search/search-page.scss

```diff
@@ -0,0 +1,44 @@
+@media (min-width: 40em) {
```

Comment:
Any chance we can use one of our existing breakpoints instead? Same goes for the other media query below.